_commands/tools/time.md

```diff
@@ -0,0 +1,123 @@
+---
+---
+
+time
+---
+`time` prints out the total time elapsed, the time used by system overhead and the time used to execute the utility.
+	
+### General Form:
+
+~~~ bash
+$ /usr/bin/time [ -apqvV ] [ -f FORMAT ] [ -o FILE ] [ --append ] [ --verbose ] [ --quiet ] [ --portability ]
+[ --format=FORMAT ] [ --output=FILE ] [ --version ] [ --help ] COMMAND [ ARGS ] 
+~~~
+
+<!--more-->
+
+### Situation
+
+If you simply want to output the time it takes to run a command
+	
+~~~ bash
+$ /usr/bin/time ./project -m < input.txt
+~~~
+
+### Meaning:
+'/usr/bin/time' calls the bash time command, './project' is the program you are trying to run,
```

Comment:
Anything code should be surrounded in \`backticks\` so it appears like `this` when you preview the site. Please make this change all across your doc.